server/src/test/scala/org/broadinstitute/dsp/workbench/welder/server/ObjectServiceSpec.scala

```diff
@@ -2,31 +2,33 @@ package org.broadinstitute.dsp.workbench.welder
 package server
 
 import java.io.File
-import java.nio.file.Paths
+import java.nio.file.Path
 
-import cats.effect.IO
 import _root_.io.chrisdavenport.log4cats.Logger
 import _root_.io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import cats.effect.IO
+import cats.effect.concurrent.Ref
+import cats.implicits._
 import io.circe.{Json, parser}
 import fs2.io
 import org.broadinstitute.dsde.workbench.google2.GcsBlobName
 import org.broadinstitute.dsde.workbench.google2.mock.FakeGoogleStorageInterpreter
 import org.broadinstitute.dsde.workbench.model.google.GcsBucketName
+import org.broadinstitute.dsp.workbench.welder.Generators._
 import org.http4s.circe.CirceEntityEncoder._
 import org.http4s.{Method, Request, Status, Uri}
 import org.scalatest.FlatSpec
-import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
-import org.broadinstitute.dsp.workbench.welder.Generators.{arbGcsBlobName, arbGcsBucketName}
 
 import scala.concurrent.ExecutionContext.global
 
-class ObjectServiceSpec extends FlatSpec with ScalaCheckPropertyChecks with WelderTestSuite {
+class ObjectServiceSpec extends FlatSpec with WelderTestSuite {
   implicit val unsafeLogger: Logger[IO] = Slf4jLogger.getLogger[IO]
-  val objectService = ObjectService(FakeGoogleStorageInterpreter, global, Paths.get("fakePath"))
+  val storageLinksCache = Ref.of[IO, Map[Path, StorageLink]](Map.empty).unsafeRunSync()
```

Comment:
can this be Ref.of[StorageLinksCache] now?

Reply:
Not really. `StorageLinksCache` is the whole `Ref`....`type StorageLinksCache = Ref[IO, Map[Path, StorageLink]]`

Reply:
ah oops